lib/cash-out/cash-out-low.js

```diff
@@ -59,6 +59,11 @@ function diff (oldTx, newTx) {
           return updatedTx['customerId'] = newTx['customerId']
         }
         return
+      case 'dispense':
```

Comment:
Could use a comment here explaining this. I guess we don't want to return a dispense diff which will trigger a dispense if the old record already indicated "dispense". But isn't this already caught on line 51?

Reply:
We don't want dispense to change from `true` to `false`, line 51 only checks for null, so false would still be legal. Will add a comment here as well.

Reply:
line 51 is this:

```
if (oldTx && _.isEqualWith(nilEqual, oldTx[fieldKey], newTx[fieldKey])) return
```

I think it's testing for equality, so if dispense is set in the new record but also in the old record, it should return a diff. Admittedly this code is too complex to be sure.

Reply:
Right, but it doesn't prevent swapping the flag.